dataplane/cmd/redbeams.go

```diff
@@ -93,6 +93,18 @@ func init() {
 							}
 						},
 					},
+					{
+						Name:   "register",
+						Usage:  "register a database server",
+						Before: cf.CheckConfigAndCommandFlagsWithoutWorkspace,
+						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlDatabaseServerRegistrationFile).AddAuthenticationFlags().AddOutputFlag().Build(),
```

Comment:
The usual way how from file is handled:
- submenu below register with name from-file
then, in submenu one mandatory option: --file (but that is more variable)
Examples:
env, credential, blueprint create